microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md

```diff
@@ -34,11 +34,19 @@ Uploads a package to update an existing draft submission of an add-on product.
 |---|---|
 | `productID` | Required.  The product ID of the product to which the package must be uploaded. |
 
-###### Request headers
+###### Request headers (v1)
```

Comment:
Is it ok to show both v1 & v2 sections/ headings/ version headers?
@sivMSFT @bhuvanapriyap

Reply:
result: implemented tabsets.  left/default tab is v2.